device-types/Juniper/PTX10001-36MR.yaml

```diff
@@ -0,0 +1,88 @@
+---
+manufacturer: Juniper
+model: PTX10001-36MR
+slug: ptx10001-36mr
+interfaces:
+  - name: re0:mgmt-0
+    type: 1000base-t
+    mgmt_only: true
+  - name: et-0/0/0
```

Comment:
Please confirm for me these are the full interface names. For instance, on a Cisco 2960 you will see `GigabitEthernet1/0/1` but you can shorthand it to `gi1/0/1`. We opt for the most complete information so we expect the interface to be named exactly as it is shown.

Can you please confirm that the devices configuration shows `et-0/0/0`?

Reply:
that is Juniper naming convention:

![image](https://github.com/netbox-community/devicetype-library/assets/50398686/60e1913c-51b4-4507-897d-0ddfe4dda4bd)

Reply:
Perfect! Thank you